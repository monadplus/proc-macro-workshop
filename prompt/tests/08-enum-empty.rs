use derive_prompt::*;

#[derive(FromPrompt)]
pub enum Choice {}

fn main() {}
