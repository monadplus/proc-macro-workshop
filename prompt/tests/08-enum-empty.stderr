error: `FromPrompt` is not supported for empty enums.
 --> tests/08-enum-empty.rs:4:10
  |
4 | pub enum Choice {}
  |          ^^^^^^
